src/components/Services/DtsList.tsx

```diff
@@ -283,17 +354,19 @@ function DtsList() {
                   )}
                   </td>
                   <td className="border-b border-[#eee] px-4 py-5 text-center dark:border-strokedark">
-                    <p
+                    
+                      <SelectUpdateState className="w-full sm:w-auto h-10 dark:text-white bg-transparent" dts={dts} ></SelectUpdateState>
+                      <WarningTimedToast message={"Error to update State"} idToast={'toast-'+dts.id} ></WarningTimedToast>
```

Comment:
the focues on component its so hard, the user needs to double click when it should be just one